corr-app/correlationDataHandler.py

```diff
@@ -0,0 +1,133 @@
+import pandas as pd
+import numpy as np
+import yfinance as yf
+
+class correlationDataHandler:
+    """
+    A data handler for the correlations
+    """
+    
+    def __init__(self, ticker_list: list):
+        self.ticker_list = ticker_list
+        
+    def build_frame(self, period) -> pd.DataFrame:
+        """
+        Get the values of interest from the list of securities from the user
+        """
+        
+        results = pd.DataFrame()
+        
+        for ticker in self.ticker_list:
```

Comment:
Ideally, we could remove some of these calcs outside of the loop so we only have to do them once across the entire dataframe (rather than re-running separately for each security). I don't know whether that provides much of a speed-up, probably not, but I kind of like that approach (to only loop over what we absolutely have to loop over). Would be interested in your thoughts